open-policy-agent/service-account-mgmt/unique_serviceaccount_per_pod/README.md

```diff
@@ -0,0 +1,295 @@
+# Policy Explainer & Testing - Unique Serviceaccount Per Pod 
+
+The policy disallows creating pods/service/deployment/deploymentconfig/replicaset (and every object that includes pod creation in any shape) that does not include a dedicated service account with name equals to the name of the pod itself.
+
+It is useful (security-wise) so that developers will get use to generate dedicated serviceAccount per micro-service, so it will be easier to audit its logs and grant elevated permissions to serviceaccounts that really requires it, without effecting multiple services at once.
+
+The required procedure to deploy the policy:
+
+1. Add the subscription for the gatekeeper operator
+2. Generate the relevant gatekeeper object CR
+3. Deploy the template & constraint yamls that define the policy
+* Note that any openshift-* & kubernetes-* default namespaces are excluded
+4. Deploy test pods & deployment object to make sure the policy works as expected
+
+`You should be logged in as cluster-admin privilaged user`
+
+## Add the subscription for the gatekeeper operator
```

Comment:
@tommeramber @michaelkotelnikov
I think it's a general guideline on OPA Gatekeeper which is irrelevant to specific policy readme and makes the README too long.

In my opinion it is better to focus on policy implementation, and example without general gatekeper configuration how-to.

Reply:
I tend to agree. I'll remove this part right away @igorkchyts @michaelkotelnikov